src/main.ts

```diff
@@ -185,6 +185,21 @@ async function main() {
 		return await handler.fetchTxArtifacts(hash);
 	});
 
+	get('/tx/fee/:extrinsic/:number', async (params) => {
```

Comment:
I think that this should be implemented as a `POST` as opposed to a `GET`. I had a brief conversation with @joepetrowski about this, @emostov, and will do the same with you :rocket: